docs/en/stack/ml/df-analytics/ml-dfa-limitations.asciidoc

```diff
@@ -147,46 +178,40 @@ majority to minority class labels in the training data. If your training data
 set is very imbalanced, consider downsampling the majority class, upsampling the 
 minority class, or gathering more data.
 
-[float]
+[discrete]
 [[dfa-inference-nested-limitation]]
-== Deeply nested objects affect {infer} performance
+=== Deeply nested objects affect {infer} performance
 
 If the data that you run inference against contains documents that have a series 
 of combinations of dot delimited and nested fields (for example: 
 `{"a.b": "c", "a": {"b": "c"},...}`), the performance of the operation might be 
 slightly slower. Consider using as simple mapping as possible for the best 
 performance profile.
 
-[float]
+[discrete]
 [[dfa-feature-importance-limitation]]
-== Analytics runtime performance may significantly slow down with feature importance computation
+=== Analytics runtime performance may significantly slow down with {feat-imp} computation
 
 For complex models (such as those with many deep trees), the calculation of 
-feature importance takes significantly more time. Feature importance is 
-calculated at the end of the analysis and therefore the job may appear to be 
-stuck at 99% for several hours.
+{feat-imp} takes significantly more time. {feat-imp-cap} is calculated at the 
```

Comment:
Calculation of feature importance is now done as results are written out and thus the computation progress is captured in the `writing_results` phase progress. We should no longer see jobs stuck at 99%.